src/java/picard/vcf/RenameSampleInVcf.java

```diff
@@ -42,7 +42,17 @@
 import java.util.EnumSet;
 
 @CommandLineProgramProperties(
-        usage = "Rename a sample within a VCF or BCF.",
+        usage = "Rename a sample within a VCF or BCF." +
+                "<br />" +
+                "<h4>Usage example:</h4>" +
+                "<pre>" +
+                "java -jar picard.jar RenameSampleInVcf \\<br />" +
+                "     -I=VCF1.vcf \\<br />" +
+                "     -O=VCFsamplerenamed.vcf \\<br />" +
+                "     -NEW_SAMPLE_NAME=Sample123" +
```

Comment:
Does this work on multi-sample VCFs? How do I specify which sample I want to rename? If I don't specify one, does it rename everything?